docker-compose.yml

```
@@ -0,0 +1,13 @@
+services:
+  redis:
+    image: redis
+    ports:
+      - "6379:6379"
+  postgres:
+    image: postgres:13
```

```suggestion
    image: postgres:14
```

That is the version I used when developing and also the version used in prod/non-prod.

It should work with 13 also but...

---

We should change the README then :)
https://github.com/Filmstund/filmstund/blob/master/README.md?plain=1#L70-L78

---

:sweat_smile: